coins/src/scripts/coingecko.ts

```diff
@@ -461,7 +462,7 @@ async function getAndStoreHourly(
       const ts = toUNIXTimestamp(price[0]);
       return !writtenTimestamps[ts];
     })
-    .map((price) =>
+    .map((price) => // TODO: why are we doing this instead of redirecting it to use cg record for price?
```

Comment:
this is related to removing stale cg mappings

Reply:
@waynebruce0x shouldnt it still check if redirect is correct and change redirects that are stale?